PTGLgraphComputation/src/proteinstructure/Chain.java

```diff
@@ -53,6 +54,18 @@
     public ArrayList<Molecule> getMolecules() { return(molecules); }
     public ArrayList<String> getHomologues() { return(homologues); }
     
+    public Boolean isRnaChain() { return(this.getMoleculeType().contains("polyribonucleotide") ? true : false); }
```

Comment:
Is not contains' return type boolean and therefore the ternary operator useless?